docs/src/main/sphinx/release/release-469.md

```diff
@@ -1,89 +1,175 @@
-# Release xyz (dd MMM 2024)
+# Release 469 (16 Jan 2025)
 
 ## General
 
-## Security
+* Add geospatial constructor {func}`ST_GeomFromKML`. ({issue}`24297`)
+* Allow deactivating the spooling protocol with the `spooling_protocol_enabled`
+  session property. ({issue}`24655`)
+* Improve stability of the cluster under load. ({issue}`24572`) 
+* Refrain from join pushdown for modified tables. ({issue}`24447`)
+* Fix parsing of negative `0x`, `0b`, `0o` long literals. ({issue}`24601`)
 
 ## Web UI
 
-## JDBC driver
+* Add support for filtering by query client tags. ({issue}`24494`)
+* Add authentication support for the Preview Web UI with the authentication
```

Comment:
Remove this one.

Reply:
ok .. not sure I agree .. we did have an entry in the 468 release notes about the preview UI .. should we remove that one as well .. I saw its also in the wrong section in 468 (general vs Web UI)